pkgs/applications/misc/soapysdr/default.nix

```
@@ -35,7 +35,7 @@ in stdenv.mkDerivation {
     # Needed for at least the remote plugin server
     for file in $out/bin/*; do
         wrapProgram "$file" \
-            --prefix SOAPY_SDR_PLUGIN_PATH : ${lib.makeSearchPath "lib/SoapySDR/modules0.6" extraPackages}
+            --prefix SOAPY_SDR_PLUGIN_PATH : ${lib.makeSearchPath "lib/SoapySDR/modules0.7" extraPackages}
```

That should be future proof:
```suggestion
            --prefix SOAPY_SDR_PLUGIN_PATH : ${with lib; makeSearchPath ("lib/SoapySDR/modules" + versions.major version + "." + versions.minor version) extraPackages}
```